compute/compute_client.go

```diff
@@ -54,7 +54,7 @@ func (c *Client) executeRequest(method, path string, body interface{}) (*http.Re
 		req.Header.Set("Content-Type", "application/oracle-compute-v3+json")
 		// Don't leak credentials in STDERR
 		if path != "/authenticate/" {
-			debugReqString = fmt.Sprintf("%s:\n %+v", debugReqString, string(reqBody))
+			debugReqString = fmt.Sprintf("%s:\nBody: %+v", debugReqString, string(reqBody))
```

Comment:
Note: this change is for consistency with the other clients that prefix `Body:` in the debug output and resolves the very strange issue that without the extra text in the Sprintf the reqBody is never output in the logs.